GenericsHomework.Tests/NodeTests.cs

```diff
@@ -0,0 +1,189 @@
+using Xunit;
+
+
+namespace GenericsHomework.Tests;
+
+public class NodeTests
+{
+    [Fact]
+    public void Node_NewNode_HasValue()
+    {
+        // Arrange
+        int val = 117;
+
+        // Act
+        Node<int> testNode = new(val);
+
+        // Assert
+
+        Assert.Equal(val, testNode.Value);
+    }
+    [Fact]
+    public void ToString_HasValue_ReturnsString()
+    {
+        // Arrange
+        int val =  42;
+
+        // Act
+        Node<int> testNode = new(val);
+
+        // Assert
+        Assert.Equal($"{val}", testNode.ToString());

```

Comment:
```suggestion
        Assert.Equal(val.ToString(), testNode.ToString());

```
Why not `.ToString()`?